CMakeLists.txt

```diff
@@ -792,10 +809,10 @@ if(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY)
 
   ### install the wrapper header file to package
   rocm_install(
-    FILES ${PROJECT_BINARY_DIR}/rccl/include/rccl.h src/include/nccl_net.h
+    FILES ${PROJECT_BINARY_DIR}/rccl/include/rccl.h src/include/nccl_net.h src/include/api_trace.h
     DESTINATION "./rccl/${CMAKE_INSTALL_INCLUDEDIR}/" )
   rocm_install(
-    FILES ${PROJECT_BINARY_DIR}/include/rccl.h src/include/nccl_net.h
+    FILES ${PROJECT_BINARY_DIR}/include/rccl.h src/include/nccl_net.h src/include/api_trace.h
```

Comment:
I’d remove these changes since the api_trace.h file doesn’t need to support the file reorg backwards compatibility.